src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import search from "./../../assets/images/search.svg";
+import "./SearchBar.scss"
+
+const SearchBar = ({handleChange}) => {
+
+  return (
+    <form className='search-bar'onSubmit= {handleChange}>
+        <input type="image" className='search-bar__image' src={search}  alt="search icon" />
```

Comment:
This is weird BUT it's good, never seen it like this before.  Only thing I would suggest is adding a label like "        <label htmlFor="search-icon" hidden></label>" See me if that is confusing.